website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Multi-Tenancy Best Practices"
+linkTitle: "Multi-Tenancy"
+weight: 4
+description: This guide describes the best practices concepts to write Operators for Multi-Tenancy solutions.
+---
+
+## Operator Behavioral Guidelines
+
+### Managing Resources
+
+#### Resource Requests
+
+In Multi-tenant clusters, administrators must consider the effects that one tenant's workloads may have on others. 
+In an effort to prevent tenants from consuming all the resources on a cluster or blocking other workloads
+from being scheduled, many production clusters will define [ResourceQuota][k8s-resources-quotas] configurations.
+
+In an effort to support multi-tenant clusters with [ResourceQuotas][k8s-resource-quotas], operators should **always** 
+include reasonable memory and CPU resource requests for safe operations and avoid negatively impacting other workloads. 
+When integrating with [OLM][olm], resource requests can be defined by modifying the`config/manager/manager.yaml` file to 
+ensure that your CSV will be build with them when you run `make bundle` as shown below:
+
+```yml
+  ...
+  # TODO(user): Configure the resources accordingly based on the project requirements.
+  # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+  resources:
+    requests:
+      cpu: 10m
+      memory: 64Mi
+    ...
+```
+
+The [ResourceQuota][k8s-resources-quotas] configuration also applies to tenant workloads that are managed by 
+your Operator. Cluster administrators will typically set a [ResourceQuota][k8s-resources-quotas] for each 
+tenant's namespace as part of the onboarding. If a LimitRange with default values has not been created 
+in each namespace and your operator creates Pods/Deployments inside the tenant namespace which do not 
+specify at least resource requests for CPU and Memory, those Pods/Deployments may fail to 
+schedule and tenants may not be able to use your Operator. 
+
+Therefore, your operator is scheduled, it should **always** apply at least resource requests for `CPU` and
```

Comment:
do you mean "therefore to ensure your operator is scheduled"?

still not sure why it wouldn't be scheduled.  The only case i can think of is that the workload could be rejected if there's no default limitrange *and* quota enforcement is turned on, and your workload doesn't specify resource requests.  i'm not sure what the behavior is in that case.

Reply:
The following are the criteria that motivate Daniels to create the original guidance:

- If a LimitRange is activated in a namespace for computing resources like CPU and memory, users must specify requests or limits for those values. Otherwise, the system may reject Pod creation. (https://kubernetes.io/docs/concepts/policy/limit-range/#overview-of-limit-range)
- If quota is enabled in a namespace for compute resources like cpu and memory, users must specify requests or limits for those values; otherwise, the quota system may reject pod creation. Hint: Use the LimitRanger admission controller to force defaults for pods that make no compute resource requirements. (https://kubernetes.io/docs/concepts/policy/resource-quotas/)

I tried to change the original idea here. Please, let me know WDYT?

Reply:
Addressed